a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -67,14 +65,11 @@ class CreteVthunderEntry(BaseDatabaseTask):
 
     def execute(self, amphora, loadbalancer, role):
         vthunder_id = uuidutils.generate_uuid()
-        self.config = self.a10_conf.get_conf()
 
-        username = self.config.get(
-            'DEFAULT', 'DEFAULT_VTHUNDER_USERNAME').replace('"', '')
-        password = self.config.get(
-            'DEFAULT', 'DEFAULT_VTHUNDER_PASSWORD').replace('"', '')
-        axapi_version = int(
-            self.config.get('DEFAULT', 'DEFAULT_AXAPI_VERSION').replace('"', ''))
+        username = CONF.VTHUNDER.DEFAULT_VTHUNDER_USERNAME
```

Comment:
Only `conf` should be capitalized. As per how it's done everywhere else in the Octavia codebase.